cfme/tests/v2v/test_v2v.py

```diff
@@ -1,9 +1,97 @@
-"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
 import pytest
+import random
 
+from cfme.automate.import_export import AutomateGitRepository
+from cfme.configure.configuration.region_settings import Tag, Category
+from cfme.infrastructure import host
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils import conf, testgen
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+from cfme.utils.update import update
 
-pytestmark = [pytest.mark.ignore_stream('5.8', '5.9')]
+pytestmark = [
+    pytest.mark.ignore_stream('5.8', '5.9'),
+    pytest.mark.meta(server_roles=['+git_owner'])
```

Comment:
This may not be necessary now as we no longer need to import git automate domain.